packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -153,7 +155,9 @@ export class JobService {
   ): Promise<CvatManifestDto> {
     const { getElementsCount } = this.createDatasetActions[requestType];
 
-    const elementsCount = await getElementsCount(requestType, dto.data);
+    const gtUrl = generateBucketUrl(dto.groundTruth, requestType);
```

Comment:
You might create an action for generateUrls that return an object like:
{
    dataset_url: string;
    gt_url: string;
    points_url: string?;
    boxes_url: string?;
}

Then you can use the urls generated here and also pass it getElementsCount or calculateJobBounty. This way you generate points_url or boxes_url twice